package.json

```diff
@@ -57,25 +57,24 @@
     "@types/configstore": "^4.0.0",
     "@types/inquirer": "^6.5.0",
     "async-mutex": "^0.2.4",
+    "bluebird": "^3.7.2",
     "chalk": "^4.0.0",
     "cli-progress": "^3.8.2",
     "commander": "^5.1.0",
     "configstore": "^5.0.1",
     "core-js": "^3.6.5",
     "encryptedfs": "^1.0.2",
-    "express": "^4.17.1",
     "futoin-hkdf": "^1.3.2",
-    "google-auth-library": "^6.0.5",
     "google-protobuf": "^4.0.0-rc.2",
     "isomorphic-git": "^1.7.4",
     "kbpgp": "^2.0.82",
-    "keybase-bot": "^3.6.1",
     "node-forge": "^0.9.1",
+    "pako": "^1.0.11",
+    "pem": "^1.14.4",
     "protobufjs": "^6.9.0",
     "readable-stream": "^3.6.0",
-    "socket.io": "^2.3.0",
+    "readline": "^1.3.0",
```

Comment:
Did you see this warning?

> Important. In node 10 there is a core module named readline. Please use linebyline instead, it is the same module just renamed:
> https://www.npmjs.com/package/readline

Reply:
Also is this something we can just copy?

Reply:
I did not see this one, but I have removed this library anyway.